### ISSUE TITLE
can't build current hackage release with ghc  7.8 RC

### ISSUE THREAD

**Issue by qnikst** [haskell-distributed/distributed-process#217]
cabal: Could not resolve dependencies:
trying: distributed-process-0.4.2 (user goal)
trying: base-4.7.0.0/installed-887... (dependency of
distributed-process-0.4.2)
next goal: rank1dynamic (dependency of distributed-process-0.4.2)
rejecting: rank1dynamic-0.1.0.2, 0.1.0.1 (conflict:
base==4.7.0.0/installed-887..., rank1dynamic => base>=4.4 && <4.7)
rejecting: rank1dynamic-0.1.0.0 (conflict: base==4.7.0.0/installed-887...,
rank1dynamic => base==4.5.*)
Dependency tree exhaustively searched.

basically the constraints need to be relaxed

**Comment by qnikst** [comment]
Fixed
